src/software/power/pins.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <Arduino.h>
```

Comment:
I don't think this constants file needs an Arduino header

Reply:
the header contains uint8_t the header is already included in other files so there shouldn't be any difference between this and <cstdint>